src/commands/createNewProject.ts

```diff
@@ -177,6 +178,31 @@ async function promotForMavenParameters(ui: IUserInterface, functionAppPath: str
     };
 }
 
+async function createJavaFunctionProject(outputChannel: OutputChannel, functionAppPath: string, ui: IUserInterface): Promise<string> {
+    // Get parameters for Maven command
+    const { groupId, artifactId, version, packageName, appName } = await promotForMavenParameters(ui, functionAppPath);
```

Comment:
Do we still need to prompt for the artifactId if we're inferring it from the original path selected?

Reply:
I think, yes we need. Because it is a property in the pom file, and can be different from the name of the project folder.